app/dashboard/models.py

```diff
@@ -4896,3 +4896,18 @@ class TribeMember(SuperModel):
         max_length=20,
         blank=True
     )
+
+    @property
+    def mutual_follow(self):
```

Comment:
tribesmembers is used for users following but also for orgs tribes seems this will show tribes orgs as following

Reply:
This is a handy method for Tribe Members, this method will be used as predicate to know if the current pair (org, profile) follow mutually.